CHANGELOG.md

```diff
@@ -358,7 +358,7 @@ API has been changed.
 ```json
 {
   partnerId: number,
-  logLevel?: string,
+  log?: ProviderLogConfigObject,
   ks?: string,
```

Comment:
Why is this marked red?

Reply:
@Yuvalke ?

Reply:
It's red because it under CHANGELOG.md I think, I deleted the changes there because it not relevant it's changelog file